src/main/java/io/github/fusionflux/portalcubed/mixin/goo/AnimalPanicMixin.java

```diff
@@ -0,0 +1,23 @@
+package io.github.fusionflux.portalcubed.mixin.goo;
+
+import com.llamalad7.mixinextras.sugar.Local;
+
+import io.github.fusionflux.portalcubed.data.tags.PortalCubedFluidTags;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.ai.behavior.AnimalPanic;
+
+import net.minecraft.world.level.BlockGetter;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+
+import java.util.function.Predicate;
+
+@Mixin(AnimalPanic.class)
+public class AnimalPanicMixin {
+	@ModifyArg(method = "lookForWater", at = @At(value = "INVOKE", target = "Lnet/minecraft/core/BlockPos;findClosestMatch(Lnet/minecraft/core/BlockPos;IILjava/util/function/Predicate;)Ljava/util/Optional;"), index = 3)
```

Comment:
all injectors that contain a full descriptor should be multi-line, fix the other one too